samples/Sentry.Samples.AspNetCore.Basic/Startup.cs

```diff
@@ -0,0 +1,70 @@
+using System;
```

Comment:
I would prefer we remove this file. The idea of `Basic` is to have the simplest possible example.
I'd prefer we continued to use a single file for this example. We can even refactor this to use top level statements. 

We can remove the WebHost example and use only your new approach

Reply:
What do you think of the current version of basic ?